pyrsa/rdm/rdms.py

```diff
@@ -107,6 +107,7 @@ def __str__(self):
     def __getitem__(self, idx):
         """
         allows indexing with []
+        and iterating over RDMs with `for rdm in rdms:`
         """
         idx = np.array(idx)
```

Comment:
if the user tries to index with something that's not a valid array index, e.g. `rdms['a']`, it will raise:

```
IndexError: arrays used as indices must be of integer (or boolean) type
```

Should it also support indexing by descriptor? And is the error message clear enough?

Reply:
This is a good question.  I also wondered about implementing slicing (e.g. `rdms[2:]`), `__setitem__` for things like `rdms[2:2] = RDMs(…)`, and `__delitem__`.  They seem like quite nice quality-of-life enhancements for objects which behave like lists of things.  But we'd need to be careful about when we return copies versus references, etc. I've made a new issue (#130) where we can discuss expanding indexing of RDMs.